Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/EditorBaseComponentMode.cpp

```diff
@@ -103,6 +111,15 @@ namespace AzToolsFramework
             return PopulateActionsImpl();
         }
 
+        AZ::Uuid EditorBaseComponentMode::GetComponentModeType() const
+        {
+            AZ_Assert(
+                false,
+                "Classes derived from EditorBaseComponentMode need to override this function and return their typeid."
+                "Example: \"return azrtti_typeid<DerivedComponentMode>();\"");
+            return AZ::Uuid();
```

Comment:
update: I think there's a `CreateNull()` function you can call here instead which I think should be preferred vs a default constructed one